Source/WebKit/Shared/WebEventConversion.cpp

```diff
@@ -359,6 +359,10 @@ class WebKit2PlatformTouchEvent : public WebCore::PlatformTouchEvent {
             return WebKit2PlatformTouchPoint(touchPoint);
         });
 
+        m_coalescedEvents = WTF::map(webEvent.coalescedEvents(), [&](auto& event) {
```

Comment:
You won't need to capture anything here.

Reply:
true; I will fix this in a follow up PR